src/widgets/star-rating/star-rating.js

```diff
@@ -0,0 +1,200 @@
+import React, { render, unmountComponentAtNode } from 'preact-compat';
```

Comment:
This widget shouldn't be here, it belongs to the `develop` branch and has been renamed to `rating-menu` (the file should be deleted).